cmd/install/install.go

```diff
@@ -150,6 +151,8 @@ func NewCommand() *cobra.Command {
 	var opts Options
 	opts.PrivatePlatform = string(hyperv1.NonePlatform)
 	opts.MetricsSet = metrics.DefaultMetricsSet
+	// TODO this needs to be updated to a multi-arch image, specifically for Arm - https://issues.redhat.com/browse/NE-1298
+	opts.ExternalDNSImage = "registry.redhat.io/edo/external-dns-rhel8:1.1.0-3"
```

Comment:
Should this use the digest instead of the tag?

Reply:
I updated this. The reason to use digest over the tag is related to work towards disconnected environment setup.